TestFoundation/TestNSURLSession.swift

```diff
@@ -21,7 +21,7 @@ class TestURLSession : XCTestCase {
 
     static var allTests: [(String, (TestURLSession) -> () throws -> Void)] {
         return [
-            ("test_dataTaskWithURL", test_dataTaskWithURL),
+/*            ("test_dataTaskWithURL", test_dataTaskWithURL),
```

Comment:
@saiHemak No, we shouldn't enable them until we get to the bottom of the CI failures. I'd suggest you have single line comments disabling each of the tests (instead of a multiline comment), with your test enabled. We should try enabling the other tests in small iterations (giving the CI enough time to reproduce the failures we've been seeing).